### PR TITLE
Mix format take 2

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: .formatter.exs [hunk 1]
```diff
@@ -1,4 +1,64 @@
 # Used by "mix format"
+
+include_patterns = ["*.exs", "{config,lib,test}/**/*.{ex,exs}"]
+
+ignore_paths = [
+  "lib/envoy/",
+  "lib/google/",
+  # TODO: Reduce this list
+  # ".formatter.exs",
```
Comment (JayH5) [inline .formatter.exs]: I should maybe just delete these commented paths...

File: .formatter.exs [hunk 2]
```diff
@@ -1,4 +1,64 @@
...
+  # "test/relay/resources_test.exs",
+  "test/relay/server_test.exs",
+  "test/relay/supervisor_test.exs",
+  "test/relay_test.exs",
+  "test/test_helper.exs"
+]
+
 [
-  inputs: ["mix.exs", "{config,lib,test}/**/*.{ex,exs}"]
+  # Unfortunately, mix format doesn't have a way to ignore paths...
```
Comment (JayH5) [inline .formatter.exs]: This is not entirely true. Another way to do things is to put extra `formatter.exs` files with `[input: []]` in each directory you want to ignore. But I don't really want to have lots of those files all over the place.

File: .formatter.exs (hunk 1 above)
Comment (jerith) [reply .formatter.exs]: Probably a good idea.

File: .formatter.exs (hunk 2 above)
Comment (jerith) [reply .formatter.exs]: Maybe rephrase the comment to include that information?
